verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -414,6 +421,25 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                                   },
                               ]
                             : []),
+                        {
```

Comment:
If you want the column to be narrower you could add padding: 0 to cell style like the field above this one.

Reply:
Will do, thanks!